models/classifier_model.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import numpy as np
+import os
+import inspect
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import accuracy_score, f1_score
+from .model import Model
+from .doc2vec_model import doc2VecModel
+
+logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s', level=logging.INFO)
+base_file_path = inspect.getframeinfo(inspect.currentframe()).filename
+base_path = os.path.dirname(os.path.abspath(base_file_path))
+project_dir_path = os.path.dirname(os.path.abspath(base_path))
+classifiers_path = os.path.join(project_dir_path, 'classifiers')
+
+class classifierModel(Model):
+    def __init__(self):
+        super().__init__()
+
+    def initialize_model(self):
+        self.model = LogisticRegression()
+
+    def train_model(self, d2v, training_vectors, training_labels):
+        logging.info("Classifier training")
+        train_vectors = doc2VecModel.get_vectors(d2v, len(training_vectors), 300, 'Train')
+        self.model.fit(train_vectors, np.array(training_labels))
+        training_predictions = self.model.predict(train_vectors)
+        logging.info('Training predicted classes: {}'.format(np.unique(training_predictions)))
+        logging.info('Training accuracy: {}'.format(accuracy_score(training_labels, training_predictions)))
+        logging.info(
+            'Training F1 score: {}'.format(f1_score(training_labels, training_predictions, average='weighted')))
+
+    def save_model(self, filename):
```

Comment:
Not implemented yet? same for loading.

Reply:
Saving and loading are implemented but removed from this PR to be contained in another one, as we agreed upon multiple self contained PRs. As per my understanding, saving models and loading them later for prediction is related to #18 that's why I have removed all code parts that are not related to #19 for now. what do you think?

Reply:
I see what you are saying, but what about removing the function headers too, then add the whole implemented functionalities in a new PR?

Reply:
Yes, makes sense, will remove headers too then